src/hugr/rewrite/simple_replace.rs

```diff
@@ -227,8 +227,7 @@ impl Rewrite for SimpleReplacement {
         }
         // 3.5. Remove all nodes in self.removal and edges between them.
         for node in &self.removal {
-            h.graph.remove_node(node.index);
-            h.hierarchy.remove(node.index);
+            h.remove_node(*node).unwrap();
```

Comment:
IIRC I was looking at simple_replace for an example of "how to do this" and then found a better way...

Reply:
Moved into #242